### ISSUE TITLE
Can this coder reproduce the results of the orgininal paper?

### ISSUE THREAD

**Issue by huangyuxin2004** [QtNN/DCA#1]
Thanks for your contribution. I'd like to confirm whether this work can reproduce the results of the paper.
I also have another question, the testing set of CNN/DAIL dataset have the separator of <t> </t>, In testing stage, do these symbols need to be removed? I'm not sure if the experimental results of this paper remove these special symbols.

**Comment by quentin-burthier** [comment]
Hi, this is also the question I want to answer and this still requires a bit of work.

Regarding the preprocessing, I have not seen any mention of differences between the preprocessing of the training and validation sets and the preprocessing of the test set in the paper.

**Comment by zide05** [comment]
I have the same question, have you reproduce the results of the paper using this code?
